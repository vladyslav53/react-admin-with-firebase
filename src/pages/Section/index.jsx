import React from 'react';

const Section = () => <h1>Section page</h1>;

export default Section;
