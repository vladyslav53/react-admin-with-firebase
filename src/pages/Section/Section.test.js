import React from 'react';
import { shallow } from 'enzyme';

import Section from '.';

describe('<Section /> rendering', () => {
  it('should render without crashing', () => {
    const component = shallow(<Section />);

    expect(component).toMatchSnapshot();
  });
});
