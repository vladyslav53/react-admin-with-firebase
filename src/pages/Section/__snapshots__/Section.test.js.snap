// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> rendering should render without crashing 1`] = `
<h1>
  Section page
</h1>
`;
